packages/mesos-modules/buildinfo.json

```diff
@@ -1,12 +1,12 @@
 {
   "requires": [
-    "mesos", 
+    "mesos",
```

Comment:
To remove noise from this patch and any automated patch building on top of it, I'd suggest to leave the EOL spaces we unfortunately get from our tooling.

Here and below.

Reply:
https://github.com/mesosphere/mesos-ci/issues/36